src/raftstore/store/msg.rs

```diff
@@ -171,6 +173,8 @@ pub enum Msg {
     MergeFail {
         region_id: u64,
     },
+
+    ValidateSSTResult(Vec<SSTMeta>),
```

Comment:
I prefer use a variable name to indicate the meaning of the result, like `ValidateSSTResult { garbage_ssts: Vec<SSTMeta> }`